### PR TITLE
fixed up snapshot-notification cli commands

### LINKED ISSUE #1568: Fix `block snapshot-get-notification-status` to work properly
FBLOCK-4193 changed the API to properly return boolean values. Need to update the manager to respect that.

### PR DESCRIPTION
**Pull request by allmightyspiff**
Fixes #1568

### CODE REVIEW

**Comment by FernandoOjeda** [comment]
TEST `snapshot-get-notification-status:`
❌ slcli block snapshot-get-notification-status 1111111
image

It seems that this request returns `"true"` string too.
image

For **null**
✔️slcli block snapshot-get-notification-status 1234567
`Enabled: Snapshots space usage threshold is enabled for volume 1234567`

For `"0"`
✔️slcli block snapshot-get-notification-status 1234567
Disabled: Snapshots space usage threshold is disabled for volume 1234567

**Comment by FernandoOjeda** [comment]
Test snapshot-set-notification:
✔️ slcli block  snapshot-set-notification --enable 1234567
✔️ slcli block  snapshot-set-notification --disable 1234567

It seems that for rest call it still support sending any string data.

File: SoftLayer/managers/storage.py [hunk 1]
```diff
@@ -19,11 +19,16 @@ class StorageManager(utils.IdentifierMixin, object):
...
     """
+
     def __init__(self, client):
         self.configuration = {}
         self.client = client
         self.resolvers = [self._get_ids_from_username]
 
+    def _get_ids_from_username(self, username):
+        """Should only be actually called from the block/file manager"""
+        return []
```
[the remark this answers is not in the document]
Comment (allmightyspiff) [reply SoftLayer/managers/storage.py:30]: I think so. This should never actually get called. This function mostly exists so I could create the storage manager test suite easier.

**Comment by allmightyspiff** [comment]
I suspect the `"true"` values are coming from volume we were testing before the API was fixed to have better support for enforcing "0"/"1" values. For now I'll leave this as is. If you can get a volume to go from `null` to `"true"` though let me know. I wasn't able to in my testing.

**Review by FernandoOjeda — APPROVED** [review]

**Pull request merged**